tests/subsys/emds/emds_flash/src/main.c

```diff
@@ -0,0 +1,699 @@
+/*
```

Comment:
The same comment is for static variables and functions. Everything that is not gone outside should be static